TAF_EasyRest/TestFramework/Tools/DBConnectionWrapper.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Npgsql;
+
+namespace TestFramework.Tools
+{
+    internal class DBConnectionWrapper
+    {
+
+        private static readonly string pg_connectionString = "Host=localhost;Username=admin;Password=12345678;Database=easyrest";
+
+        static void ShowHeaders(NpgsqlDataReader reader)
+        {
+            var columnNames = Enumerable.Range(0, reader.FieldCount)
+                        .Select(reader.GetName)
+                        .ToArray();
+            foreach (var column in columnNames) { Console.Write($"{column}\t"); }
+            Console.WriteLine();
+        }
+
+        static void ShowRows(NpgsqlDataReader reader)
```

Comment:
please specify the access modifier explicitly

Reply:
done.